pfitzpaddy/ngm-reportHub
# Change form Target location

test cases:
1. Add new location
2. Remove location
3. Edit Location ( change province, change site name, etc )
4. Activities check
5. opening new (add new target location on a new project)
6. checking locations in reports, correct location info saved
